master broken got ppc64le platform

I'm trying to build https://github.com/ContinuumIO/anaconda-recipes/tree/master/sqlite on `linux-ppc64le` using conda-build master 6f13867d4208040b6ddfa17b1eb2c102b032b7de, and I get:
```
...
Traceback (most recent call last):
  File "/home/ilan/minonda/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/home/ilan/conda-build/conda_build/cli/main_build.py", line 242, in main
    execute(sys.argv[1:])
  File "/home/ilan/conda-build/conda_build/cli/main_build.py", line 234, in execute
    already_built=None, config=config)
  File "/home/ilan/conda-build/conda_build/api.py", line 77, in build
    need_source_download=need_source_download, config=config)
  File "/home/ilan/conda-build/conda_build/build.py", line 1074, in build_tree
    config=recipe_config)
  File "/home/ilan/conda-build/conda_build/build.py", line 819, in build
    tarcheck.check_all(tmp_path)
  File "/home/ilan/conda-build/conda_build/tarcheck.py", line 92, in check_all
    x.correct_subdir()
  File "/home/ilan/conda-build/conda_build/tarcheck.py", line 85, in correct_subdir
    " got {1}".format(subdir, info['subdir']))
AssertionError: Incorrect subdir in package - expecting linux-ppc64le, got linux-64
```
This was working fine using conda-build 1.19.2

--- a/conda_build/build.py
+++ b/conda_build/build.py
@@ -483,8 +483,7 @@ def create_env(prefix, specs, config, clear_cache=True):
                 for folder in locked_folders:
                     if not os.path.isdir(folder):
                         os.makedirs(folder)
-                    lock = get_lock(join(folder, '.conda_lock'),
-                                                 timeout=config.timeout)
+                    lock = get_lock(folder, timeout=config.timeout)
                     if not folder.endswith('pkgs'):
                         update_index(folder, config=config, lock=lock, could_be_mirror=False)
                     locks.append(lock)
@@ -669,80 +668,78 @@ def build(m, config, post=None, need_source_download=True, need_reparse_in_env=F
 
         print("Package:", m.dist())
 
-        with get_lock(join(config.build_folder, ".conda_lock"),
-                                   timeout=config.timeout):
-            # get_dir here might be just work, or it might be one level deeper,
-            #    dependening on the source.
-            src_dir = config.work_dir
-            if isdir(src_dir):
-                print("source tree in:", src_dir)
+        # get_dir here might be just work, or it might be one level deeper,
+        #    dependening on the source.
+        src_dir = config.work_dir
+        if isdir(src_dir):
+            print("source tree in:", src_dir)
+        else:
+            print("no source - creating empty work folder")
+            os.makedirs(src_dir)
+
+        rm_rf(config.info_dir)
+        files1 = prefix_files(prefix=config.build_prefix)
+        for pat in m.always_include_files():
+            has_matches = False
+            for f in set(files1):
+                if fnmatch.fnmatch(f, pat):
+                    print("Including in package existing file", f)
+                    files1.discard(f)
+                    has_matches = True
+            if not has_matches:
+                log.warn("Glob %s from always_include_files does not match any files", pat)
+        # Save this for later
+        with open(join(config.croot, 'prefix_files.txt'), 'w') as f:
+            f.write(u'\n'.join(sorted(list(files1))))
+            f.write(u'\n')
+
+        # Use script from recipe?
+        script = m.get_value('build/script', None)
+        if script:
+            if isinstance(script, list):
+                script = '\n'.join(script)
+
+        if isdir(src_dir):
+            if on_win:
+                build_file = join(m.path, 'bld.bat')
+                if script:
+                    build_file = join(src_dir, 'bld.bat')
+                    with open(build_file, 'w') as bf:
+                        bf.write(script)
+                import conda_build.windows as windows
+                windows.build(m, build_file, config=config)
             else:
-                print("no source - creating empty work folder")
-                os.makedirs(src_dir)
+                build_file = join(m.path, 'build.sh')
 
-            rm_rf(config.info_dir)
-            files1 = prefix_files(prefix=config.build_prefix)
-            for pat in m.always_include_files():
-                has_matches = False
-                for f in set(files1):
-                    if fnmatch.fnmatch(f, pat):
-                        print("Including in package existing file", f)
-                        files1.discard(f)
-                        has_matches = True
-                if not has_matches:
-                    log.warn("Glob %s from always_include_files does not match any files", pat)
-            # Save this for later
-            with open(join(config.croot, 'prefix_files.txt'), 'w') as f:
-                f.write(u'\n'.join(sorted(list(files1))))
-                f.write(u'\n')
-
-            # Use script from recipe?
-            script = m.get_value('build/script', None)
-            if script:
-                if isinstance(script, list):
-                    script = '\n'.join(script)
-
-            if isdir(src_dir):
-                if on_win:
-                    build_file = join(m.path, 'bld.bat')
+                # There is no sense in trying to run an empty build script.
+                if isfile(build_file) or script:
+                    with path_prepended(config.build_prefix):
+                        env = environ.get_dict(config=config, m=m)
+                    env["CONDA_BUILD_STATE"] = "BUILD"
+                    work_file = join(config.work_dir, 'conda_build.sh')
                     if script:
-                        build_file = join(src_dir, 'bld.bat')
-                        with open(build_file, 'w') as bf:
+                        with open(work_file, 'w') as bf:
                             bf.write(script)
-                    import conda_build.windows as windows
-                    windows.build(m, build_file, config=config)
-                else:
-                    build_file = join(m.path, 'build.sh')
-
-                    # There is no sense in trying to run an empty build script.
-                    if isfile(build_file) or script:
-                        with path_prepended(config.build_prefix):
-                            env = environ.get_dict(config=config, m=m)
-                        env["CONDA_BUILD_STATE"] = "BUILD"
-                        work_file = join(config.work_dir, 'conda_build.sh')
-                        if script:
-                            with open(work_file, 'w') as bf:
-                                bf.write(script)
-                        if config.activate:
-                            if isfile(build_file):
-                                data = open(build_file).read()
-                            else:
-                                data = open(work_file).read()
-                            with open(work_file, 'w') as bf:
-                                bf.write('source "{conda_root}activate" "{build_prefix}" &> '
-                                         '/dev/null\n'.format(conda_root=root_script_dir +
-                                                              os.path.sep,
-                                                              build_prefix=config.build_prefix))
-                                bf.write(data)
+                    if config.activate:
+                        if isfile(build_file):
+                            data = open(build_file).read()
                         else:
-                            if not isfile(work_file):
-                                copy_into(build_file, work_file, config.timeout)
-                        os.chmod(work_file, 0o766)
+                            data = open(work_file).read()
+                        with open(work_file, 'w') as bf:
+                            bf.write('source "{conda_root}activate" "{build_prefix}" &> '
+                                        '/dev/null\n'.format(conda_root=root_script_dir +
+                                                            os.path.sep,
+                                                            build_prefix=config.build_prefix))
+                            bf.write(data)
+                    else:
+                        if not isfile(work_file):
+                            copy_into(build_file, work_file, config.timeout)
+                    os.chmod(work_file, 0o766)
 
-                        if isfile(work_file):
-                            cmd = [shell_path, '-x', '-e', work_file]
-                            # this should raise if any problems occur while building
-                            _check_call(cmd, env=env, cwd=src_dir)
+                    if isfile(work_file):
+                        cmd = [shell_path, '-x', '-e', work_file]
+                        # this should raise if any problems occur while building
+                        _check_call(cmd, env=env, cwd=src_dir)
 
     if post in [True, None]:
         if post:
@@ -830,7 +827,7 @@ can lead to packages that include their dependencies.""" % meta_files))
 
 def clean_pkg_cache(dist, timeout):
     cc.pkgs_dirs = cc.pkgs_dirs[:1]
-    locks = [get_lock(join(folder, ".conda_lock"), timeout=timeout) for folder in cc.pkgs_dirs]
+    locks = [get_lock(folder, timeout=timeout) for folder in cc.pkgs_dirs]
     with ExitStack() as stack:
         for lock in locks:
             stack.enter_context(lock)
@@ -874,116 +871,115 @@ def test(m, config, move_broken=True):
 
     clean_pkg_cache(m.dist(), config.timeout)
 
-    with get_lock(join(config.build_folder, ".conda_lock"), timeout=config.timeout):
-        tmp_dir = config.test_dir
-        if not isdir(tmp_dir):
-            os.makedirs(tmp_dir)
-        create_files(tmp_dir, m, config)
-        # Make Perl or Python-specific test files
-        if m.name().startswith('perl-'):
-            pl_files = create_pl_files(tmp_dir, m)
-            py_files = False
-            lua_files = False
-        else:
-            py_files = create_py_files(tmp_dir, m)
-            pl_files = False
-            lua_files = False
-        shell_files = create_shell_files(tmp_dir, m, config)
-        if not (py_files or shell_files or pl_files or lua_files):
-            print("Nothing to test for:", m.dist())
-            return
+    tmp_dir = config.test_dir
+    if not isdir(tmp_dir):
+        os.makedirs(tmp_dir)
+    create_files(tmp_dir, m, config)
+    # Make Perl or Python-specific test files
+    if m.name().startswith('perl-'):
+        pl_files = create_pl_files(tmp_dir, m)
+        py_files = False
+        lua_files = False
+    else:
+        py_files = create_py_files(tmp_dir, m)
+        pl_files = False
+        lua_files = False
+    shell_files = create_shell_files(tmp_dir, m, config)
+    if not (py_files or shell_files or pl_files or lua_files):
+        print("Nothing to test for:", m.dist())
+        return
 
-        print("TEST START:", m.dist())
+    print("TEST START:", m.dist())
 
-        get_build_metadata(m, config=config)
-        specs = ['%s %s %s' % (m.name(), m.version(), m.build_id())]
+    get_build_metadata(m, config=config)
+    specs = ['%s %s %s' % (m.name(), m.version(), m.build_id())]
 
-        # add packages listed in the run environment and test/requires
-        specs.extend(ms.spec for ms in m.ms_depends('run'))
-        specs += ensure_list(m.get_value('test/requires', []))
+    # add packages listed in the run environment and test/requires
+    specs.extend(ms.spec for ms in m.ms_depends('run'))
+    specs += ensure_list(m.get_value('test/requires', []))
 
-        if py_files:
-            # as the tests are run by python, ensure that python is installed.
-            # (If they already provided python as a run or test requirement,
-            #  this won't hurt anything.)
-            specs += ['python %s*' % environ.get_py_ver(config)]
-        if pl_files:
-            # as the tests are run by perl, we need to specify it
-            specs += ['perl %s*' % environ.get_perl_ver(config)]
-        if lua_files:
-            # not sure how this shakes out
-            specs += ['lua %s*' % environ.get_lua_ver(config)]
+    if py_files:
+        # as the tests are run by python, ensure that python is installed.
+        # (If they already provided python as a run or test requirement,
+        #  this won't hurt anything.)
+        specs += ['python %s*' % environ.get_py_ver(config)]
+    if pl_files:
+        # as the tests are run by perl, we need to specify it
+        specs += ['perl %s*' % environ.get_perl_ver(config)]
+    if lua_files:
+        # not sure how this shakes out
+        specs += ['lua %s*' % environ.get_lua_ver(config)]
 
-        create_env(config.test_prefix, specs, config=config)
+    create_env(config.test_prefix, specs, config=config)
 
-        with path_prepended(config.test_prefix):
-            env = dict(os.environ.copy())
-            env.update(environ.get_dict(config=config, m=m, prefix=config.test_prefix))
-            env["CONDA_BUILD_STATE"] = "TEST"
+    with path_prepended(config.test_prefix):
+        env = dict(os.environ.copy())
+        env.update(environ.get_dict(config=config, m=m, prefix=config.test_prefix))
+        env["CONDA_BUILD_STATE"] = "TEST"
 
-        if not config.activate:
-            # prepend bin (or Scripts) directory
-            env = prepend_bin_path(env, config.test_prefix, prepend_prefix=True)
-
-            if on_win:
-                env['PATH'] = config.test_prefix + os.pathsep + env['PATH']
-
-        for varname in 'CONDA_PY', 'CONDA_NPY', 'CONDA_PERL', 'CONDA_LUA':
-            env[varname] = str(getattr(config, varname) or '')
-
-        # Python 2 Windows requires that envs variables be string, not unicode
-        env = {str(key): str(value) for key, value in env.items()}
-        suffix = "bat" if on_win else "sh"
-        test_script = join(tmp_dir, "conda_test_runner.{suffix}".format(suffix=suffix))
-
-        with open(test_script, 'w') as tf:
-            if config.activate:
-                ext = ".bat" if on_win else ""
-                tf.write('{source} "{conda_root}activate{ext}" "{test_env}" {squelch}\n'.format(
-                    conda_root=root_script_dir + os.path.sep,
-                    source="call" if on_win else "source",
-                    ext=ext,
-                    test_env=config.test_prefix,
-                    squelch=">nul 2>&1" if on_win else "&> /dev/null"))
-                if on_win:
-                    tf.write("if errorlevel 1 exit 1\n")
-            if py_files:
-                tf.write("{python} -s {test_file}\n".format(
-                    python=config.test_python,
-                    test_file=join(tmp_dir, 'run_test.py')))
-                if on_win:
-                    tf.write("if errorlevel 1 exit 1\n")
-            if pl_files:
-                tf.write("{perl} {test_file}\n".format(
-                    perl=config.test_perl,
-                    test_file=join(tmp_dir, 'run_test.pl')))
-                if on_win:
-                    tf.write("if errorlevel 1 exit 1\n")
-            if lua_files:
-                tf.write("{lua} {test_file}\n".format(
-                    lua=config.test_lua,
-                    test_file=join(tmp_dir, 'run_test.lua')))
-                if on_win:
-                    tf.write("if errorlevel 1 exit 1\n")
-            if shell_files:
-                test_file = join(tmp_dir, 'run_test.' + suffix)
-                if on_win:
-                    tf.write("call {test_file}\n".format(test_file=test_file))
-                    if on_win:
-                        tf.write("if errorlevel 1 exit 1\n")
-                else:
-                    # TODO: Run the test/commands here instead of in run_test.py
-                    tf.write("{shell_path} -x -e {test_file}\n".format(shell_path=shell_path,
-                                                                       test_file=test_file))
+    if not config.activate:
+        # prepend bin (or Scripts) directory
+        env = prepend_bin_path(env, config.test_prefix, prepend_prefix=True)
 
         if on_win:
-            cmd = ['cmd.exe', "/d", "/c", test_script]
-        else:
-            cmd = [shell_path, '-x', '-e', test_script]
-        try:
-            subprocess.check_call(cmd, env=env, cwd=tmp_dir)
-        except subprocess.CalledProcessError:
-            tests_failed(m, move_broken=move_broken, broken_dir=config.broken_dir, config=config)
+            env['PATH'] = config.test_prefix + os.pathsep + env['PATH']
+
+    for varname in 'CONDA_PY', 'CONDA_NPY', 'CONDA_PERL', 'CONDA_LUA':
+        env[varname] = str(getattr(config, varname) or '')
+
+    # Python 2 Windows requires that envs variables be string, not unicode
+    env = {str(key): str(value) for key, value in env.items()}
+    suffix = "bat" if on_win else "sh"
+    test_script = join(tmp_dir, "conda_test_runner.{suffix}".format(suffix=suffix))
+
+    with open(test_script, 'w') as tf:
+        if config.activate:
+            ext = ".bat" if on_win else ""
+            tf.write('{source} "{conda_root}activate{ext}" "{test_env}" {squelch}\n'.format(
+                conda_root=root_script_dir + os.path.sep,
+                source="call" if on_win else "source",
+                ext=ext,
+                test_env=config.test_prefix,
+                squelch=">nul 2>&1" if on_win else "&> /dev/null"))
+            if on_win:
+                tf.write("if errorlevel 1 exit 1\n")
+        if py_files:
+            tf.write("{python} -s {test_file}\n".format(
+                python=config.test_python,
+                test_file=join(tmp_dir, 'run_test.py')))
+            if on_win:
+                tf.write("if errorlevel 1 exit 1\n")
+        if pl_files:
+            tf.write("{perl} {test_file}\n".format(
+                perl=config.test_perl,
+                test_file=join(tmp_dir, 'run_test.pl')))
+            if on_win:
+                tf.write("if errorlevel 1 exit 1\n")
+        if lua_files:
+            tf.write("{lua} {test_file}\n".format(
+                lua=config.test_lua,
+                test_file=join(tmp_dir, 'run_test.lua')))
+            if on_win:
+                tf.write("if errorlevel 1 exit 1\n")
+        if shell_files:
+            test_file = join(tmp_dir, 'run_test.' + suffix)
+            if on_win:
+                tf.write("call {test_file}\n".format(test_file=test_file))
+                if on_win:
+                    tf.write("if errorlevel 1 exit 1\n")
+            else:
+                # TODO: Run the test/commands here instead of in run_test.py
+                tf.write("{shell_path} -x -e {test_file}\n".format(shell_path=shell_path,
+                                                                    test_file=test_file))
+
+    if on_win:
+        cmd = ['cmd.exe', "/d", "/c", test_script]
+    else:
+        cmd = [shell_path, '-x', '-e', test_script]
+    try:
+        subprocess.check_call(cmd, env=env, cwd=tmp_dir)
+    except subprocess.CalledProcessError:
+        tests_failed(m, move_broken=move_broken, broken_dir=config.broken_dir, config=config)
 
     print("TEST END:", m.dist())
 

--- a/conda_build/config.py
+++ b/conda_build/config.py
@@ -12,7 +12,7 @@ import sys
 import time
 
 from .conda_interface import string_types, binstar_upload
-from .conda_interface import root_dir, root_writable, cc, bits, platform
+from .conda_interface import root_dir, root_writable, cc, subdir, platform
 
 from .utils import get_build_folders, rm_rf
 
@@ -105,7 +105,7 @@ class Config(object):
                   Setting('verbose', False),
                   Setting('debug', False),
                   Setting('timeout', 90),
-                  Setting('bits', bits),
+                  Setting('arch', subdir.split('-')[-1]),
                   Setting('platform', platform),
                   Setting('set_build_id', True),
                   Setting('disable_pip', False)
@@ -124,14 +124,14 @@ class Config(object):
         if self.platform == 'noarch':
             return self.platform
         else:
-            return "-".join([self.platform, str(self.bits)])
+            return "-".join([self.platform, str(self.arch)])
 
     @subdir.setter
     def subdir(self, value):
         values = value.split('-')
         self.platform = values[0]
         if len(values) > 1:
-            self.bits = values[1]
+            self.arch = values[1]
 
     @property
     def croot(self):

--- a/conda_build/environ.py
+++ b/conda_build/environ.py
@@ -235,7 +235,7 @@ def conda_build_vars(prefix, config):
         'CONDA_BUILD': '1',
         'PYTHONNOUSERSITE': '1',
         'CONDA_DEFAULT_ENV': config.build_prefix,
-        'ARCH': str(config.bits),
+        'ARCH': str(config.arch),
         'PREFIX': prefix,
         'SYS_PREFIX': sys.prefix,
         'SYS_PYTHON': sys.executable,
@@ -383,7 +383,7 @@ def unix_vars(prefix):
 
 
 def osx_vars(compiler_vars, config):
-    OSX_ARCH = 'i386' if config.bits == 32 else 'x86_64'
+    OSX_ARCH = 'i386' if config.arch == 32 else 'x86_64'
     compiler_vars['CFLAGS'] += ' -arch {0}'.format(OSX_ARCH)
     compiler_vars['CXXFLAGS'] += ' -arch {0}'.format(OSX_ARCH)
     compiler_vars['LDFLAGS'] += ' -arch {0}'.format(OSX_ARCH)
@@ -398,7 +398,7 @@ def osx_vars(compiler_vars, config):
 
 def linux_vars(compiler_vars, prefix, config):
     compiler_vars['LD_RUN_PATH'] = prefix + '/lib'
-    if config.bits == 32:
+    if config.arch == 32:
         compiler_vars['CFLAGS'] += ' -m32'
         compiler_vars['CXXFLAGS'] += ' -m32'
     return {}

--- a/conda_build/index.py
+++ b/conda_build/index.py
@@ -19,8 +19,7 @@ def read_index_tar(tar_path, config, lock=None):
     """ Returns the index.json dict inside the given package tarball. """
 
     if not lock:
-        lock = get_lock(join(os.path.dirname(tar_path), ".conda_lock"),
-                                     timeout=config.timeout)
+        lock = get_lock(os.path.dirname(tar_path), timeout=config.timeout)
     with ExitStack() as stack:
         stack.enter_context(lock)
         t = tarfile.open(tar_path)
@@ -77,7 +76,7 @@ def update_index(dir_path, config, force=False, check_md5=False, remove=True, lo
         os.makedirs(dir_path)
 
     if not lock:
-        lock = get_lock(join(dir_path, ".conda_lock"))
+        lock = get_lock(dir_path)
 
     with lock:
         if force:

--- a/conda_build/utils.py
+++ b/conda_build/utils.py
@@ -111,7 +111,7 @@ def copy_into(src, dst, timeout=90, symlinks=False, lock=None):
                 src_folder = os.getcwd()
 
         if not lock:
-            lock = get_lock(join(src_folder, ".conda_lock"), timeout=timeout)
+            lock = get_lock(src_folder, timeout=timeout)
         with lock:
             # with each of these, we are copying less metadata.  This seems to be necessary
             #   to cope with some shared filesystems with some virtual machine setups.
@@ -190,7 +190,7 @@ def merge_tree(src, dst, symlinks=False, timeout=90, lock=None):
                       "{2}".format(src, dst, existing[0]))
 
     if not lock:
-        lock = get_lock(join(src, ".conda_lock"), timeout=timeout)
+        lock = get_lock(src, timeout=timeout)
     with lock:
         copytree(src, dst, symlinks=symlinks)
 
@@ -201,13 +201,14 @@ def merge_tree(src, dst, symlinks=False, timeout=90, lock=None):
 _locations = {}
 
 
-def get_lock(lock_file, timeout=90):
+def get_lock(folder, timeout=90, filename=".conda_lock"):
     global _locations
-    location = os.path.abspath(os.path.normpath(lock_file))
-    if not os.path.isdir(os.path.dirname(location)):
-        os.makedirs(os.path.dirname(location))
+    location = os.path.abspath(os.path.normpath(folder))
+    if not os.path.isdir(location):
+        os.makedirs(location)
     if location not in _locations:
-        _locations[location] = filelock.SoftFileLock(location, timeout)
+        _locations[location] = filelock.SoftFileLock(os.path.join(location, filename),
+                                                     timeout)
     return _locations[location]
 
 

--- a/conda_build/utils.py
+++ b/conda_build/utils.py
@@ -25,7 +25,7 @@ import filelock
 from .conda_interface import md5_file, unix_path_to_win, win_path_to_unix
 from .conda_interface import PY3, iteritems
 from .conda_interface import linked
-from .conda_interface import bits, root_dir
+from .conda_interface import root_dir
 
 from conda_build.os_utils import external
 
@@ -515,7 +515,8 @@ def create_entry_point(path, module, func, config):
             if 'debug' in packages_names:
                 fo.write('#!python_d\n')
             fo.write(pyscript)
-        copy_into(join(dirname(__file__), 'cli-%d.exe' % bits), path + '.exe', config.timeout)
+        copy_into(join(dirname(__file__), 'cli-%d.exe' % config.arch),
+                  path + '.exe', config.timeout)
     else:
         with open(path, 'w') as fo:
             fo.write('#!%s\n' % config.build_python)

--- a/conda_build/utils.py
+++ b/conda_build/utils.py
@@ -515,7 +515,7 @@ def create_entry_point(path, module, func, config):
             if 'debug' in packages_names:
                 fo.write('#!python_d\n')
             fo.write(pyscript)
-        copy_into(join(dirname(__file__), 'cli-%d.exe' % config.arch),
+        copy_into(join(dirname(__file__), 'cli-{}.exe'.format(config.arch)),
                   path + '.exe', config.timeout)
     else:
         with open(path, 'w') as fo:

--- a/tests/test_config.py
+++ b/tests/test_config.py
@@ -53,23 +53,23 @@ def test_create_config_with_subdir():
 
 def test_set_platform(config):
     config.platform = 'steve'
-    bits = config.bits
-    assert config.subdir == 'steve-' + str(bits)
+    arch = config.arch
+    assert config.subdir == 'steve-' + str(arch)
 
 
 def test_set_subdir(config):
     config.subdir = 'steve'
-    bits = config.bits
-    assert config.subdir == 'steve-' + str(bits)
+    arch = config.arch
+    assert config.subdir == 'steve-' + str(arch)
     assert config.platform == 'steve'
 
     config.subdir = 'steve-128'
     assert config.subdir == 'steve-128'
     assert config.platform == 'steve'
-    assert config.bits == '128'
+    assert config.arch == '128'
 
 
 def test_set_bits(config):
-    config.bits = 128
+    config.arch = 128
     assert config.subdir == config.platform + '-' + str(128)
-    assert config.bits == 128
+    assert config.arch == 128

